main.mk

```diff
@@ -41,6 +41,14 @@ endif
 %.txt: %.xml
 	$(xml2rfc) $< -o $@ --text
 
+%.html: %.jrxml rfc2629xslt/rfc2629.xslt
+    $(xsltproc) rfc2629xslt/rfc2629.xslt $< > $@
+
+%.txt: %.jrxml rfc2629xslt/clean-for-DTD.xslt
```

Comment:
If you formulate this as two separate rules (one for translating .jrxml to .cleaned.xml; one for translating to text, then you don't need to include the rm step because make will do it for you.